<launch>
    <node pkg="pal_parallel_gripper_wrapper" name="gripper_grasping" type="gripper_grasping.py" output="screen">
        <!-- Dynamic reconfigure parameters -->
        <param name="grasper/max_position_error" type="double" value="0.002"/>
        <param name="grasper/timeout" type="double" value="5.0"/>
        <param name="grasper/closing_time" type="double" value="2.0"/>
        <param name="grasper/rate" type="int" value="5"/>
    </node>
</launch>
